frontend/viewer/src/lib/hooks/is-mobile.svelte.ts

```diff
@@ -0,0 +1,9 @@
+import {MediaQuery} from 'svelte/reactivity';
+
+const MOBILE_BREAKPOINT = 768;
```

Comment:
could this value be shared with the tailwind config somehow?

Reply:
A shared constants.ts file for example?